app/services/strategy_testing_service.py

```
@@ -0,0 +1,752 @@
+        self.min_backtest_days = 90      # Minimum 90 days for valid backtest
+        self.min_trades_for_significance = 30  # Minimum trades for statistical significance
+        self.confidence_level = 0.95     # 95% confidence for statistical tests
+    
+    async def run_strategy_backtest(
+        self,
+        strategy_function: str,
+        symbol: str,
+        start_date: datetime,
+        end_date: datetime,
+        initial_capital: Decimal = Decimal("10000"),
+        parameters: Optional[Dict[str, Any]] = None
```

Ruff (0.12.2)

Unused method argument: `parameters`

(ARG002)